packages/cli/cloud/src/services/cli-api.ts

```diff
@@ -181,6 +183,23 @@ export async function cloudApiFactory(
       }
     },
 
+    async getProject({ name }): Promise<AxiosResponse<unknown>> {
+      try {
+        const response = await axiosCloudAPI.get(`/projects/${name}`);
+
+        if (response.status !== 200) {
+          throw new Error("Error fetching project's info from the server.");
```

Comment:
Might be a bit too confusing for users to talk about "the server". 🤔 
```suggestion
          throw new Error("There was a problem retrieving your project details. Please try again later.");
```

Reply:
I was being consistent with the rest of the error messages which also use this wording. But I agree.